docs/core-concepts/networking.md

```diff
@@ -57,26 +56,26 @@ Each host Node has a total of four different network addresses:
 
   The public network address is resolved at startup using the `http://whatismyip.akamai.com` service, or it can be configured using `KONTENA_PUBLIC_IP`.
 
-  The Node's public addess can be used to connect to network services exposed on that host Node, including ports published by any Kontena Service Container that has been scheduled to run on that Node, including any instance of the Kontena Load Balancer.
+  The Node's public addess can be used to connect to network services exposed on that host Node. These include ports published by any Kontena Service Container that has been scheduled to run on that Node, including any instance of the Kontena Load Balancer.
   The Node's public address is also used for the Weave control and data plane connections between Nodes.
-  The Weave control and data plane ports are the only publically exposed services on a host Node in the default configuration.
+  The Weave control and data plane ports are the only publically exposed services on a host Node under the default Kontena configuration.
```

Comment:
This is about the default configuration of the kontena-agent on the host nodes.

CoreOS will presumably also expose at least SSH.